lib/analytics.rb

```diff
@@ -0,0 +1,20 @@
+class Analytics
+  attr_reader :dmv_data_service
+
+  def initialize()
+    @dmv_data_service = DmvDataService.new
+  end
+
+  def most_popular_car
+    popular_vehicles = []
+    factory = VehicleFactory.new
+    registration = dmv_data_service.wa_ev_registrations
+    all_vehicles = factory.create_vehicles(registration)
+    specific_vehicles = all_vehicles.map do |vehicle|
+      vehicle.make + " " + vehicle.model
+    end
+    specific_vehicles.group_by(&:itself).values.max_by(&:size).first
+  end
```

Comment:
Great job making a separate class for this. This most_popular_car method is great overall. I think line 16 could probably be broken apart into different pieces so we can more clearly see what is going on. I've never seen the `itself` keyword before, very interesting!